docs/design/autoware-interfaces/ad-api/list/api/route/lanelet/notice.md

```diff
@@ -0,0 +1,14 @@
+# /api/route/lanelet/notice
+
+- Method: notification
+- Type: [autoware_ad_api_msgs/msg/LaneletRouteNotice](../../../../types/autoware_ad_api_msgs/msg/lanelet_route_notice.md)
+
+## Description
+
+Get the route with the waypoint segments in lanelet format. It is empty if route is not set.
```

Comment:
Why do you not use ROS service?

Reply:
If using the service, polling is needed to detect route updates.

Reply:
Ah, do you want to notify clients when the route is cleared?
If so, let's write the reason in the documentation.

Reply:
OK. Add the comment to the working branch.
https://github.com/isamu-takagi/autoware.universe/blob/feature/ad-api/prototype/common/autoware_ad_api_msgs/README.md